airflow-ctl/src/airflowctl/api/client.py

```diff
@@ -145,7 +145,7 @@ class Client(httpx.Client):
 
     def __init__(self, *, base_url: str, token: str, **kwargs: Any):
         auth = BearerAuth(token)
-        kwargs["base_url"] = f"{base_url}/public"
+        kwargs["base_url"] = f"{base_url}"
```

Comment:
```suggestion
        kwargs["base_url"] = f"{base_url}/api/v2"
```

I made this change in the other PR I limked in the below comment